schemas/HPXMLBaseElements.xsd

```diff
@@ -5063,30 +5063,7 @@
 					<xs:attribute name="dataSource" type="DataSource"/>
 				</xs:complexType>
 			</xs:element>
-			<xs:element minOccurs="0" name="StormWindow">
-				<xs:complexType>
-					<xs:sequence>
-						<xs:group ref="SystemInfo"/>
-						<xs:element minOccurs="0" name="GlazingMaterial" type="GlazingMaterial"/>
-						<xs:element minOccurs="0" name="GlassType" type="GlassType"/>
-						<xs:element minOccurs="0" name="FrameType" type="WindowFrameType"/>
-						<xs:element minOccurs="0" name="Location" type="StormLocation"/>
-						<xs:element minOccurs="0" name="Operable" type="HPXMLBoolean">
```

Comment:
@shorowit I removed this deprecated element for v4.0 according to its descriptions. Does it look reasonable?

Reply:
No, we were proposing to add this deprecation warning for v4 to give time for users to adjust. Now you're proposing to remove it immediately!

I suggest making your changes based on the `master` branch rather than the `v4_0` branch. The latter includes proposals that have not yet been discussed/approved.

Reply:
Sure, I will add it back and switch to base on master branch